pennylane/interfaces/execution.py

```diff
@@ -21,7 +21,7 @@
 
 # pylint: disable=import-outside-toplevel,too-many-arguments,too-many-branches,not-callable
 # pylint: disable=unused-argument,unnecessary-lambda-assignment,inconsistent-return-statements,
-# pylint: disable=too-many-statements, invalid-unary-operand-type
+# pylint: disable=too-many-statements, invalid-unary-operand-type, function-redefined

```

Comment:
The amount of pylint warnings that need to be disabled for this file to work is making me sweat.

Reply:
And that's not even getting into the number of sourcery complaints I get locally.